### ISSUE TITLE
Failed to connect

### ISSUE THREAD

**Issue by Lockie85** [jcgoette/baby_buddy_homeassistant#146]
Hi,

I'm unable to add my Baby Buddy Addon within Home Assistant. Any idea as to why it keeps failing. I believe got the correct IP, port and API. But is there anyway I can confirm this?

**Comment by CrashyNL** [comment]
Same problem here. 

I'm using Nabu Casa, any help would be much appreciated!

**Comment by jcgoette** [comment]
@Lockie85 , are you using Nabu Casa as well?

If so, both you and @CrashyNL should read through https://github.com/jcgoette/baby_buddy_homeassistant/issues/130 and https://github.com/jcgoette/baby_buddy_homeassistant/issues/140.

**Comment by germangaguilar** [comment]
Same situation here. I have Nabu, but I'm using muy local IP as host... don't know how to get the integration logs, since I can't install the integration.

Add-on works just fine with the url: http://myIP:8123/6888888_baby_buddy/ingress

I'm trying with:

Host: http://myIP
Port: 8123
Path: /6888888_baby_buddy/ingress
Key: Apikey

**Comment by Nachtom** [comment]
I had the same issue. Tried everything suggested here and in those other linked threads, nothing worked. After whole evening spent on this, I figured out it was just one stupid letter that caused me all these troubles. One! Just **switching from https:// to http://** in front of host IP helped.

I could even see that my configuration is kinda correct - because if I put it wrong port/host it fails immediately, but with "almost" correct one it processed it for few seconds before fail. So in case somebody else ends up with the same situation this is another possible fix. :)
